core/src/main/scala/org/apache/spark/storage/PushBasedFetchHelper.scala

```diff
@@ -181,8 +183,9 @@ private class PushBasedFetchHelper(
 
       override def onFailure(shuffleId: Int, shuffleMergeId: Int, reduceId: Int,
           exception: Throwable): Unit = {
-        logError(s"Failed to get the meta of push-merged block for ($shuffleId, $reduceId) " +
-          s"from ${req.address.host}:${req.address.port}", exception)
+        logError(log"Failed to get the meta of push-merged block for " +
```

Comment:
It's not related to this PR, do we need `shuffleMergeId` in the log?

Reply:
Well let's just keep it.